vlib/orm/README.md

```diff
@@ -11,6 +11,7 @@
 - `[primary]` sets the field as the primary key
 - `[unique]` sets the field as unique
 - `[unique: 'foo']` adds the field to a unique group
+- `[nonull]` set the field as not null
```

Comment:
Is it `not null` or `cannot be null`?

Reply:
`not null` is the SQL syntax. I think that it is clear as it is in the PR, but perhaps the explanation can be added too, as a separate sentence after that.

Reply:
Then the attribute is mis-named.  It should be `[notnull]`.  `no null` and `not null` mean 2 different things.